aioesphomeapi/model.py

```diff
@@ -886,6 +917,15 @@ class MediaPlayerCommand(APIIntEnum):
     STOP = 2
     MUTE = 3
     UNMUTE = 4
+    TOGGLE = 5
+    VOLUME_UP = 6
+    VOLUME_DOWN = 7
+    ENQUEUE = 8
```

Comment:
How do you expect this one to be used?

Reply:
Don't know, but it is in "esphome/components/media_player/media_player.h".  So I added it to normalize with that work.